File: WebFt30/src/app/core/store/models/ft30/network-event.ts
import { ChannelEvent } from './ft-enums';

export class NetworkEvent {
  eventId!: number;
  registeredAt!: Date;
  rtuTitle!: string;
  rtuId!: string;
  isRtuAvailable!: boolean;
  onMainChannel!: ChannelEvent;
  onReserveChannel!: ChannelEvent;
}

File: Backend/src/Iit/Iit.Fibertest.Graph/DomainEntities/ExternalEvents/NetworkEvent.cs
using Iit.Fibertest.Dto;
// ReSharper disable InconsistentNaming

namespace Iit.Fibertest.Graph
{
    [Serializable]
    public class NetworkEvent : INotificationEvent
    {
        public int Ordinal { get; set; }

        public DateTime EventTimestamp { get; set; }
        public Guid RtuId { get; set; }
        public ChannelEvent OnMainChannel { get; set; }
        public ChannelEvent OnReserveChannel { get; set; }

        public bool IsRtuAvailable { get; set; }

    }
}